templates/terraform/modules/environment/variables.tf

```diff
@@ -181,3 +181,27 @@ variable "ci_user_name" {
   type        = string
   description = "CI user name" 
 }
+
+variable "cache_instance_type" {
+  type        = string
+  default     = "cache.t2.micro"
+  description = "Elastic cache instance type"
+}
+
+variable "cache_cluster_size" {
+  type        = number
+  default     = 1
+  description = "Number of nodes in cluster"
+}
+
+variable "cache_store" {
+  type        = string
+  default     = "none"
```

Comment:
should the default be `""`?

Reply:
Ah nevermind, we are already using `"none"` for other stuff. Fine to stick with that.